src/connectors/CVMFSConnector.ts

```diff
@@ -0,0 +1,198 @@
+import SlurmConnector from "./SlurmConnector";
+import { slurm, executableManifest } from "../types";
+import { containerConfigMap } from "../../configs/config";
+import * as config from "../../config.json";
+
+class CVMFSConnector extends SlurmConnector {
+  /**
+   * Connects the singularity container to the hpc enivironment
+   */
+  private volumeBinds: { [keys: string]: string } = {};
+
+  public isContainer = true;
+  /**
+   * Executes specified command within specified image
+   *
+   * @param{string} image - docker image
+   * @param{string} cmd - command to be executed
+   * @param{slurm} config_slurm - slurm configuration
+   */
+  execCommandWithinImage(image: string, cmd: string, config_slurm: slurm) {
+    cmd = `srun --mpi=pmi2 ${
+      config.cvmfs_path
```

Comment:
I think for all of these `config.cvmfs_path` you can just put `singcvmfs` because we can add the location of the executable to the $PATH.

Reply:
Yup thanks ! Removed it in the latest commit.